WalletWasabi.Gui/Tabs/WalletManager/LoadWalletViewModel.cs

```diff
@@ -318,19 +310,13 @@ private bool TrySetWalletStates()
 				IsWalletSelected = SelectedWallet != null;
 				CanTestPassword = IsWalletSelected;
 
-				if (Global.WalletService is null)
+				if (Global.WalletServiceManager.AnyWalletService())
 				{
-					IsWalletOpened = false;
-
-					// If not busy loading.
-					// And wallet is selected.
-					// And no wallet is opened.
-					CanLoadWallet = !IsBusy && IsWalletSelected;
+					IsWalletOpened = true;
 				}
 				else
 				{
-					IsWalletOpened = true;
-					CanLoadWallet = false;
+					IsWalletOpened = false;
 				}
```

Comment:
Can be converted to conditional expression `IsWalletOpened = Global.WalletServiceManager.AnyWalletService();`

Reply:
Then I would loose the notifying property?

Reply:
Which notifying property? sorry I don't get what you are asking.
This is just a syntax change.
```
if (Global.WalletServiceManager.AnyWalletService())
{
	IsWalletOpened = true;
}
else
{
	IsWalletOpened = false;
}
```
Can be written in one line like this:
`IsWalletOpened = Global.WalletServiceManager.AnyWalletService();`

Reply:
oh!